Tests/Packet++Test/PacketExamples/TcpPacketWithOptions2.dat

```diff
@@ -1 +1 @@
-080027191c7830469a23fbfa08004500003d4f4c40003b064798d4c7ca090a0000060050ac80b829cb98e977158680184ed2f2aa00000101080ad3f03998000300950049454e44ae426082
\ No newline at end of file
+080027191c7830469a23fbfa0800450000414f4c40003b064794d4c7ca090a0000060050ac80b829cb98e977158690184ed2dea100000101080ad3f0399800030095010303020049454e44ae426082
```

Comment:
what's the difference here?

Reply:
I modified test `TcpTests::TcpPacketCreation` which uses this file as PCAP data. I wanted to test both deprecated API and new API in one test case. I added another TCP option "WINDOW" using new API and I had to update this PCAP data.